Update ROADMAP document.

pauljamescleary opened the pull request:
Organized into large features, and added plan implementation
and availability.

britneywright commented on ROADMAP.md:
```diff
@@ -1,30 +1,57 @@
+implementation can hopefully be contributed by members in the open source community, or perhaps the demand shuffles
+our priorities.  These include:
+
+1. DNS SEC - There is no first-class support for DNS SEC.  That feature set is being defined.
```
What is DNS SEC? DNS Security?

britneywright commented on ROADMAP.md:
```diff
@@ -1,30 +1,57 @@
+
+1. DNS SEC - There is no first-class support for DNS SEC.  That feature set is being defined.
 1. Record meta data - VinylDNS will allow the "tagging" of DNS records with arbitrary key-value pairs
 1. DNS GSLB - Integration with various GSLB vendors for common DNS GSLB configurations
```
I would write out GSLB at least once here for those who might not be familiar with what it means.

pauljamescleary commented:
@britneywright unclear on how granular the `ROADMAP` would become.  Hopefully this just stays to high-level features.  I imagine that a new automation tool would be valuable to see here.

Changes to existing tools I would not expect to see in the `ROADMAP` as they are too low level.

We should have a CHANGELOG, minimally we need to be disciplined about Github releases to make sure we capture all of the changes since the last release.  This holds true for the tooling repos.
